modules/events.js

```diff
@@ -0,0 +1,47 @@
+import Books from './books.js';
+
+const $title = document.querySelector('.title');
+const $author = document.querySelector('.author');
+const $addButton = document.querySelector('.addbutton');
+const $error = document.querySelector('.error');
+const $addPage = document.querySelector('.addpage');
+const $contactPage = document.querySelector('.acontact');
+const $listPage = document.querySelector('.listpage');
+const $booksSection = document.querySelector('.books');
+const $newSection = document.querySelector('.newbooks');
+const $contactSection = document.querySelector('.contactinfo');
+const bookObj = new Books();
+
+export default function events() {
+  $addButton.addEventListener('click', () => {
+    if ($title.value !== '' && $author.value !== '') {
+      bookObj.add($title.value.trim(), $author.value.trim());
+      $error.style.display = 'none';
+      $title.value = '';
+      $author.value = '';
+    } else {
+      $error.style.display = 'block';
+      setTimeout(() => {
+        $error.style.display = 'none';
+      }, 1500);
+    }
+  });
+
+  $addPage.addEventListener('click', () => {
+    $booksSection.style.display = 'none';
+    $contactSection.style.display = 'none';
+    $newSection.style.display = 'block';
+  });
+
+  $contactPage.addEventListener('click', () => {
+    $booksSection.style.display = 'none';
+    $contactSection.style.display = 'block';
+    $newSection.style.display = 'none';
+  });
+
+  $listPage.addEventListener('click', () => {
+    $booksSection.style.display = 'block';
+    $contactSection.style.display = 'none';
+    $newSection.style.display = 'none';
+  });
+}
```

Comment:
[OPTIONAL] Well done here 💯 but for better code readability and to respect ES6 syntax consider using the arrow syntax for functions. Here you would have something:
```
const events = () => {
   // content of the function
};

export default events;
```
I moved the export at the end to fix some errors 💯 anyway I hope this helps. 🤓

Reply:
@didierganthier Thanks for your feedback!